README.md

```diff
@@ -79,6 +81,22 @@ while True:
 rdr.cleanup()
 ```
 
+To retrieve tag UID in either 4 or 7 byte form, you can use:
+
+```python
+from pirc522 import RFID
+rdr = RFID()
+
+while True:
+  rdr.wait_for_tag()
+  uid = rdr.read_id(as_number = True)
+  if uid is not None:
+    print(f'UID: {uid:X}')
```

Comment:
This is a python3.6+ way for formatting a string, I just get an error testing with kivypie which only has 2.7 and 3.4
swapped it for `print('UID: {:X}'.format(uid_number))`